utils/colors.js

```diff
@@ -0,0 +1,7 @@
+const colors = {
+  black: '#151616',
```

Comment:
maybe it would be better to export these separately

```suggestion
export const black = '#151616',
```